prod-devel-rcar-virtio.yaml

```diff
@@ -1,4 +1,4 @@
-desc: "Xen-Troops development setup for Renesas RCAR Gen3 hardware (depends on the private meta-xt-vhost, see README.md)"
```

Comment:
So, there are no mentions in README?

Reply:
No. All vhost-specific text in README was regarding access to the proprietary repo, and was already removed by https://github.com/xen-troops/meta-xt-prod-devel-rcar/pull/132/files